src/web-rtc-client.js

```diff
@@ -139,6 +139,7 @@ export default class WebRTCClient extends Emitter {
       if (isMobile()) {
         this._fixLocalDescription(session, 'answer');
       }
+      // @TODO: useful ? replaced by the Answer-Mode header. Should be deprecated
```

Comment:
Yes is useful, the alert-info header is classically used for the auto answer mode in phone.